### PR TITLE
Fix namespace of macros

### PR DESCRIPTION
**Pull request by mdesmet**
## Overview

- Description:
- Related issue(s):
- Related code pull request(s):
- Related link(s):

## Checklist

- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [ ] `README.md` updated and added information about my change
- [ ] `CHANGELOG.md` updated and added information about my change

### CODE REVIEW

File: dbt/include/trino/macros/crossdb/any_value.sql [hunk 1]
```diff
@@ -1,3 +1,3 @@
-{% macro default__any_value(expression) -%}
+{% macro trino__any_value(expression) -%}
```
Comment (findinpath) [inline dbt/include/trino/macros/crossdb/any_value.sql:1]: Why are we doing this change?

**Comment by findinpath** [comment]
Even if this is a small / insignificant change, let's wait on a review before merging it.

cc @hovaesco

**Comment by mdesmet** [comment]
The case where this could impact things is the following. If a user installs multiple dbt adapters and that adapter does not override the default macro, I'm not sure if this macro would override the core macro and potentially be used for other adapter's executions. Maybe it's not an issue as the adapter is only loaded if the profile refers to the adapter but we would have to check dbt core's code to know this for sure.

File: dbt/include/trino/macros/crossdb/any_value.sql (hunk 1 above)
Comment (mdesmet) [reply dbt/include/trino/macros/crossdb/any_value.sql:1]: Any adapter-specific code should not reimplement the default but override the default by providing the`<adapter>__macro` implementation if necessary.

**Comment by mdesmet** [comment]
> Maybe it's not an issue as the adapter is only loaded if the profile refers to the adapter but we would have to check dbt core's code to know this for sure.

It's not an issue the adapter is loaded based on the type property in the dbt profile. So it's impossible for two adapters to be loaded at the same time. See https://github.com/dbt-labs/dbt-core/blob/32415e365961cb453e2a4dd5f5002461848f1202/core/dbt/adapters/factory.py#L56-L73

**Pull request merged**